src/ui/camera.js

```diff
@@ -884,22 +889,24 @@ class Camera extends Evented {
         if (tr.mercatorFromTransition && zoom < halfZoomTransition) {
             tr.setProjection({name: 'globe'});
             tr.zoom = zoom;
-            return this._cameraForBounds(tr, p0, p1, bearing, options);
+            return this._cameraForBounds(tr, p0, p1, bearing, pitch, options);
         }
 
-        return {center, zoom, bearing};
+        return {center, zoom, bearing, pitch};
     }
 
     /**
      * Pans and zooms the map to contain its visible area within the specified geographical bounds.
-     * This function will also reset the map's bearing to 0 if bearing is nonzero.
+     * This function will also reset the map's pitch and bearing to 0 if bearing is nonzero.
```

Comment:
Yes, good catch! I'll confirm that it's still the behavior and update.

Reply:
I've simply removed that statement as it can be confusing, and the default values added [here](https://github.com/mapbox/mapbox-gl-js/pull/12367/files/da175abb77fa75af50e3a2a18181f7c5ca727431#diff-6e31e9c70d2c897b3a306e5a6718a4b6698e8b5eb3a11729da97fe21ccc45261R908-R909) should be enough to understand the behavior.